pandas/plotting/_matplotlib/core.py

```diff
@@ -100,7 +105,7 @@ def __init__(
         data,
         kind=None,
         by=None,
-        subplots=False,
+        subplots: bool | Sequence[Sequence[str]] = False,
```

Comment:
I _think_ in general, pandas uses `Union` instead of `|`, would be nicer to align with other similar cases in the codebase.

Reply:
ah, okay! sorry about that, i was not aware of such update in pre-commit hook.

Reply:
emm, interesting, still see `Union` used in codebase. maybe @simonjayhawkins or @MarcoGorelli knows better about this change in the precommit hook?